pkg/clusterfile/clusterfile.go

```diff
@@ -30,17 +34,28 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	configMapNamespace = "kube-system"
+	configMapDataName  = "Clusterfile"
+	configMapName      = "sealer-clusterfile"
+)
+
 type Interface interface {
```

Comment:
Shall we save the application.json into k8s too?

Reply:
i will do it in the another PR.